### DISCUSSION TITLE
Is v2.4.4rc4 stable to be used in production and when could be the potential next release?

### DISCUSSION THREAD

**Discussion by noorul** [abhinavsingh/proxy.py#1344] [Q&A]
Is v2.4.4rc4 stable to be used in production and when could be the potential next release?

**Comment by abhinavsingh** [comment]
@noorul Yep, the develop branch is typically pretty stable.   We deploy `proxy.py` directly out of `develop` branch and haven't had any issues ever.   Agreed, we should make more regular releases.   Will try to cut one over this weekend.   Thanks for bringing this to my attention.
